### ISSUE TITLE
[Error] E517: No buffers were wiped out: bwipeout! 3

### ISSUE THREAD

**Issue by tamago324** [statiolake/vim-quickrun-runner-nvimterm#1]
- `nvim --version`: NVIM v0.5.0-dev+1321-gc57a85d53
- Operating system/version: Manjaro

**Describe the bug**

When I close the Result window with `:q` and run QuickRun again, I get the following error.

```
Error detected while processing function quickrun#command#execute[10]..quickrun#run[10]..108[10]..101:
line   18:
E517: No buffers were wiped out: bwipeout! 3
```

**To Reproduce using `nvim -u mini.vim -i NONE`**

```vim
filetype plugin indent on
set rtp+=~/.config/nvim/.plugged/vim-quickrun
set rtp+=~/.config/nvim/.plugged/vim-quickrun-runner-nvimterm
language messages en_US.utf8

let g:quickrun_config = {
\   '_': {
\       'runner': 'nvimterm',
\   },
\}
```

1. `nvim -u mini.vim -i NONE`
2. `:e sample.py`
3. `:QuickRun`
4. `:wincmd w`
5. `:q`
6. `:QuickRun`

An error occurs.

**Expected behavior**

Closing the result window and running it again will not cause an error.

**Screenshots**

Thank you!

https://user-images.githubusercontent.com/16581287/118480675-87953700-b74d-11eb-991c-ce64ba881d60.mp4

**Comment by statiolake** [comment]
Thanks for your detailed report! It reproduced and I fixed it via #2.
Check if problem persists. If it does, please reopen this issue.

**Comment by tamago324** [comment]
Confirmed that it's been fixed!
Thanks for the quick response!
